README.md

```diff
@@ -0,0 +1,11 @@
+# SciTools/.github
+
+Organisational level things such as the SciTools README and organisation workflows
+
+## Possible uses
```

Comment:
This is entirely up to you, extremely subjective!

I'd possibly have a "current uses" heading with a brief overview of what the project does? I'm just thinking if this gets bigger, it'd be helpful having an index.

e.g. (very rough)
```
## Current uses
 - [Peloton](./peloton): A project board containing issues and PRs from SciTools repos, with predefined filters for use of keeping outstanding issues/PRs.
```

This could negate the use of the other readme, my b.

Reply:
I can absolutely understand that argument. 

I'd argue, much like the whatsnew, it's just something that the reviewer would hopefully catch, and I think would add enough value that it's worth the effort.